browser_tests/tests/mobileBaseline.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { comfyPageFixture as test } from '../fixtures/ComfyPage'
+import { expect } from '@playwright/test'
+
+test.describe('Mobile Baseline Snapshots', () => {
```

Comment:
Slight preference for just tagging the suite:
```suggestion
test.describe('Mobile Baseline Snapshots', {
  tag: '@mobile',
}, () => {
```